4-fleetscope/modules/env_baseline/mcg.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  fleet_membership_re = "//gkehub.googleapis.com/(.*)$"
+}
+
+resource "google_project_service" "mci" {
+  project = var.project_id
+  service = "multiclusteringress.googleapis.com"
+}
+
+resource "google_project_service" "mcsd" {
+  project = var.project_id
+  service = "multiclusterservicediscovery.googleapis.com"
+}
+
+resource "google_gke_hub_feature" "feature" {
+  name = "multiclusteringress"
+  depends_on = [
+    google_project_service.mci,
+    google_project_service.mcsd,
+  ]
+  location = "global"
+  project  = var.project_id
+  spec {
+    multiclusteringress {
+      config_membership = regex(local.fleet_membership_re, var.cluster_membership_ids[0])[0]
```

Comment:
Do we know if it will also accept the entire `"//gkehub.googleapis.com/projects/foo-proj/locations/global/memberships/bar` format?  I'd prefer to avoid regex when possible.

Reply:
i tried it yesterday, it does not accept the entire "//gkehub.googleapis.com/projects/foo-proj/locations/global/memberships/bar"

Reply:
Got it - Figured that might be the case.  Thanks!